src/components/structures/TagPanel.js

```diff
@@ -71,6 +74,17 @@ const TagPanel = React.createClass({
         dis.dispatch(GroupActions.fetchJoinedGroups(this.context.matrixClient));
     },
 
+    onClientSync(syncState, prevState) {
+        if (this.unmounted) return;
```

Comment:
this is technically unnecessary, since the listener is removed in WillUnmount.

it's harmless though